modules/lotamePanoramaIdSystem.js

```diff
@@ -168,6 +184,7 @@ export const lotamePanoramaIdSubmodule = {
    * @returns {IdResponse|undefined}
    */
   getId(config, consentData, cacheIdObj) {
+    cookieDomain = lotamePanoramaIdSubmodule.findRootDomain();
```

Comment:
delayed assignment of cookieDomain till here is just to save the call if execution short-circuited for other reasons?

Reply:
This is the earliest this code can run as there is not "init" phase and when the module is first activated, it is too early to use the cookie utils. And yes, we don't want to do it every time we write a cookie either.